hints/hint_distribution.py

```diff
@@ -235,7 +235,7 @@ def start(self, logic: Logic, always_hints: list, sometimes_hints: list):
 
         region_barren, nonprogress = self.logic.get_barren_regions()
         for zone in region_barren:
-            if "Silent Realm" in zone:
+            if "Silent Realm" in zone or "Flooded Faron Woods":
```

Comment:
```suggestion
            if "Silent Realm" in zone or zone == "Flooded Faron Woods":
```
Currently every region is skipped since `"Flooded Faron Woods"` casts to true

Reply:
Fixed :p

Reply:
python moment